5-Cookies/server/services/MongoDBSerivce.ts

```diff
@@ -0,0 +1,37 @@
+import { errorHandler } from "../utils/errorHandler";
+import { MongoDB } from "../utils/MongoDB";
+
+require("dotenv").config("karate-kid-kickstart/5-Cookies/server/.env");
+const mongoose = require("mongoose");
+
+export class MongoDBService extends MongoDB {
+  connection;
+  constructor() {
+    super();
+  }
+  async connect() {
+    try {
+      const url: string = `${process.env.DB_HOST}`;
```

Comment:
how about `String(process.env.DB_HOST)`? sounds a bit less "magic"y to me, not a big deal tho